src/components/products/ProductCard.tsx

```diff
@@ -1,26 +1,27 @@
-import { component$, mutable } from '@builder.io/qwik';
-import { Link } from '@builder.io/qwik-city';
+import { component$ } from '@builder.io/qwik';
 import Price from './Price';
 
 export default component$(
 	({ productAsset, productName, slug, priceWithTax, currencyCode }: any) => {
 		return (
-			<Link className="flex flex-col" href={`/products/${slug}`}>
+			<a className="flex flex-col" href={`/products/${slug}`}>
```

Comment:
I used <Link> because I prefer SPA behaviour, I don't like the whole refresh feeling

Reply:
My mistake. I changed when I was getting the node_modules error on dev.ssr. I'll change it back in the morning and clean everything up.

Reply:
Great 👌

Reply:
Fixed!